kola/tests/locksmith/locksmith.go

```diff
@@ -83,13 +84,16 @@ func locksmithCluster(c cluster.TestCluster) error {
 	for _, m := range machs {
 		worker := func(c context.Context) error {
 			// XXX: stop sshd so checkmachine verifies correctly if reboot worked
-			// XXX: run locksmithctl under systemd-run so our current connection doesn't drop suddenly
-			cmd := "sudo systemctl stop sshd.socket; sudo systemd-run --quiet --on-active=2 --no-block locksmithctl send-need-reboot"
+			// XXX: run under systemd-run so our current connection doesn't drop suddenly
+			cmd := "sudo systemd-run --quiet --on-active=5 --no-block sh -exc 'systemctl stop sshd.socket && exec locksmithctl send-need-reboot'"
```

Comment:
alternatively you can check for and ignore https://godoc.org/golang.org/x/crypto/ssh#ExitMissingError